tests/PriorityMapTest.php

```diff
@@ -24,14 +24,14 @@ public function testToArrayDesc()
             'k4' => 'v4',
         );
 
-        foreach($list as $key => $value) {
+        foreach ($list as $key => $value) {
             $this->assertEquals(key($expectedArray), $key);
             $this->assertEquals(current($expectedArray), $value);
 
             next($expectedArray);
         }
 
-        if(key($expectedArray)) {
+        if (key($expectedArray)) {
```

Comment:
it was not necessary to fix my fauls, thank you )))))

Reply:
I just use the ```fix-style``` to follow the PSR-2 style.